recipes/lemon/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "lemon" %}
+{% set version = "1.3.1" %}
+{% set sha256 = "71b7c725f4c0b4a8ccb92eb87b208701586cf7a96156ebd821ca3ed855bad3c8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://lemon.cs.elte.hu/pub/sources/lemon-{{ version  }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - glpk_win.diff    # [win]
+
+build:
+  number: 0
+  features:
+    - vc9           # [win and py27]
+    - vc14          # [win and py>=35]
+
+requirements:
+  build:
+    - python        # [win]
+    - toolchain
+    - cmake
+    - glpk
+    - vc 9          # [win and py27]
+    - vc 14         # [win and py>=35]
+    - coincbc       # [unix]
+    - doxygen
+    - graphviz
+    - ghostscript   # [unix]
+  run:
+    - glpk
+    - vc 9          # [win and py27]
+    - vc 14         # [win and py>=35]
+    - coincbc       # [unix]
+
+test:
+  commands:
+    - test -f $PREFIX/include/lemon/glpk.h                 # [unix]
+    - if not exist %LIBRARY_INC%\\lemon\\glpk.h exit 1     # [win]
```

Comment:
Also some library file tests would be good, but that should be pretty straightforward.